sqlmesh/core/config/connection.py

```diff
@@ -1734,6 +1734,8 @@ def get_catalog(self) -> t.Optional[str]:
         exclude=(ConnectionConfig, BaseDuckDBConnectionConfig),
     )
 }
+CONNECTION_CONFIG_TO_TYPE["tsql"] = CONNECTION_CONFIG_TO_TYPE["mssql"]
+CONNECTION_CONFIG_TO_TYPE["sqlserver"] = CONNECTION_CONFIG_TO_TYPE["mssql"]
```

Comment:
These should probably be reverted, given [this](https://github.com/tobymao/sqlglot/pull/4666) was closed.